src/ui/QuoteMessageInput/index.tsx

```diff
@@ -72,7 +72,9 @@ export default function QuoteMessageInput({
           {isGifMessage(fileMessage) && stringSet.QUOTE_MESSAGE_INPUT__FILE_TYPE_GIF}
           {isUserMessage(replyingMessage as UserMessage) && (replyingMessage as UserMessage).message}
           {getUIKitMessageType(replyingMessage) === UIKitMessageTypes.FILE && getMessageFirstFileName(fileMessage)}
-          {isMultipleFilesMessage(replyingMessage as CoreMessageType) && getMessageFirstFileName(fileMessage)}
+          {
+            isMultipleFilesMessage(replyingMessage as CoreMessageType)
+            && `${(fileMessage as MultipleFilesMessage).fileInfoList.length} Photos`}
```

Comment:
{0 or 2 >=} photos / 1 photo <-- I guess we need to handle the pluralization as well?

Reply:
Ah we don't need to handle the single file case cause it's not `multipleFileMessage` 😄